main.go

```diff
@@ -129,11 +131,24 @@ func main() {
 
 	cableEngineSyncer.Run(stopCh)
 
+	err = subv1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fatal(cableEngineSyncer, "Error adding submariner types to the scheme: %v", err)
```

Comment:
nit: Any specific reason for not using "klog.Fatal" in the new code?

Reply:
`fatal` cals `klog.Fatal`.